styles/components/LoggedOut.module.less

```diff
@@ -0,0 +1,73 @@
+/**
+* Copyright (c) 2014, Tidepool Project
```

Comment:
This appears to be a new file, but the year is 2014. Should be it 2023?

Reply:
Ah, yeah, just used another file to spawn this one. We actually generally don't include the license and copyright block on new files. I can get rid of it.